ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -29,6 +29,10 @@ if(_source_files)
 
   # Get include paths for added targets
   set(_all_include_dirs "")
+  if(DEFINED AMENT_CMAKE_CPPCHECK_INCLUDE_DIRS)
+    list_append_unique(_all_include_dirs ${AMENT_CMAKE_CPPCHECK_INCLUDE_DIRS})
```

Comment:
`list(APPEND ...)` should be sufficient here since the list is empty and the variable provided by the user which has control about avoiding duplicates in the first place.

Reply:
650de6975bb269a2716343bbf39901c92b889dd9